bson/raw_bson.py

```diff
@@ -13,6 +13,62 @@
 # limitations under the License.
 
 """Tools for representing raw BSON documents.
+
+Inserting and Retrieving RawBSONDocuments
+=========================================
+
+Example: Moving a document between different databases/collections
+
+.. testsetup::
+  from pymongo import MongoClient
+  client = MongoClient("localhost", 27017, document_class=RawBSONDocument)
+  client.drop_database('db')
+  client.drop_database('replica_db')
+
+.. doctest::
+
+  >>> import bson
+  >>> from pymongo import MongoClient
+  >>> from bson.raw_bson import RawBSONDocument
+  >>> client = MongoClient("localhost", 27017, document_class=bson.)
+  >>> db = client.db
+  >>> result = db.test.insert_many([
+  ...  {'a': 1},
+  ...  {'b': 1},
+  ...  {'c': 1},
+  ...  {'d': 1}])
+  >>> assert result.acknowledged
+  >>> replica_db = client.replica_db
+  >>> for doc in db.test.find():
+  ...    print(f"raw document: {doc.raw}")
+  ...    print(f"decoded document: {bson.decode(doc.raw)}")
+  ...    result = replica_db.test.insert_one(doc)
+  ...    assert result.acknowledged
```

Comment:
I think we can ignore result here:
```
replica_db.test.insert_one(doc)
```

The `acknowledged` property isn't expected to be checked very often and it's not very relevant to this example.

Reply:
removed!